src/Apps/Article/Components/ArticleAd/ArticleAd.tsx

```diff
@@ -30,21 +37,40 @@ export const ArticleAd: FC<ArticleAdProps> = memo(({ unit, size, ...rest }) => {
         mx="auto"
         bg="black10"
       >
-        <Box
-          ref={ref as any}
-          width="100%"
-          height="100%"
-          style={{
-            transformOrigin: "top left",
-            transform: `scale(${scale})`,
-          }}
-        >
-          <AdSlot
-            adUnit={unit}
-            sizes={[[width, height]]}
-            onSlotIsViewable={displayedAd}
-          />
-        </Box>
+        {ready && (
+          <>
+            {isDestinationAllowed("Google Ads") ? (
```

Comment:
is there any way we could roll `ready` up into `isDestinationAllowed` so that additional bit of (necessary) logic is transparent to future devs?

Reply:
Actually, that's not desirable. Note what's actually happening here. When we aren't `ready` we don't display anything`, when we are we display either the button or the ad. That's because we don't want to display `"Manage cookies"` to US users. There's 3 states: loading, requires consent, content

Reply:
Its just a bit hard to know how to use ready, should other areas be needed in the future. The description provided above doesn't quite translate into the code at a glance. 

If this is necessary complexity then no problem. And maybe it could be resolved by updating the name `ready` to something less general.

Reply:
And actually, `ready` _is_ rolled into that function as well. In some cases that might be fine. I just expose `ready` for when it's not.

Reply:
Cool, nothing blocking here as long as you think it's clear enough for the future 👍

Reply:
I'll add a README